tock/tock/settings/base.py

```diff
@@ -55,6 +55,7 @@
                 'django.template.context_processors.tz',
                 'django.contrib.messages.context_processors.messages',
                 'django.template.context_processors.request',
+                'tock.context_processors.version_url',
```

Comment:
Double-check me on this, but I'm not sure this is the optimal order for context processors. If I recall correctly the order of them does matter, and I think you want your somewhere higher. But I may be mis-remembering.

Reply:
Good catch @tbaxter-18f, the order definitely matters, but I can't remember off the top of my head either.  [The documentation on them](https://docs.djangoproject.com/en/1.10/topics/templates/#context-processors) (accounting for the version of Django currently running for Tock) is a little light, but I might be looking in the wrong place, too.

Asking in our #django channel in Slack is a good place to follow up on this to see if someone else remembers or at least knows where to look. :-)

Reply:
From what I gathered from the docs, it seems like context processor's order matters in the sense that it's possible for them to overwrite variables found in other context processors. I'm going to keep the order here, but I'm going to export the dictionary with a `x_tock_` prefix to avoid any possible name collisions with internal Django stuff. Thanks!